lms/templates/ccx/coach_dashboard.html

```diff
@@ -23,8 +23,8 @@
 <section class="container">
   <div class="instructor-dashboard-wrapper-2">
         <main id="main" aria-label="Content" tabindex="-1">
-        <section class="instructor-dashboard-content-2" id="ccx-coach-dashboard-content">
-          <h1>${_("CCX Coach Dashboard")}</h1>
+        <section class="instructor-dashboard-content-2" id="ccx-coach-dashboard-content" aria-labelledby="header-ccx-dashboard">
+          <h2 class="hd hd-2" id="header-ccx-dashboard">${_("CCX Coach Dashboard")}</h2>
```

Comment:
I'd like to keep it consistent with other Instructor Dashboard-level sections. The logo of the page is the H1 so the next highest heading should be an H2.